clients/drcachesim/common/trace_entry.h

```diff
@@ -272,9 +272,17 @@ typedef enum {
     TRACE_MARKER_TYPE_CPU_ID,
 
     /**
-     * The marker value contains the function id defined by the user in the
-     * -record_function (and -record_heap_value if -record_heap is specified)
-     * option.
+     * The marker value contains the function id for functions traced by the user via
+     * the -record_function (and -record_heap_value if -record_heap is specified)
+     * option.  The mapping from this numeric ID to a library-qualified symbolic name
+     * is recorded during tracing in a file "funclist.log" whose format is described by
+     * the drmemtrace_get_funclist_path() function's documentation.
+     *
+     * This marker is also used to record parameter values for certain system calls such
+     * as for #OFFLINE_FILE_TYPE_KERNEL_SCHED.  These use large identifiers equal to
```

Comment:
Do we need to provide more details about these system call identifiers? Normalized vs not normalized etc.  That they are present only for LINUX.

Reply:
Added a sentence pointing at  dr_register_pre_syscall_event()'s docs for normalized numbers for the source of the number used.

Reply:
For only Linux: it points at the KERNEL_SCHED which lists only futex so I think that's enough.